Swift/Collection.swift

```diff
@@ -283,11 +282,15 @@ public final class Collection : CollectionChangeObservable, Indexable {
     
     // MARK: Internal
     
-    init(impl: CBLCollection) {
+    var db: Database {
+        return _db
+    }
+    
+    init(_ impl: CBLCollection) {
```

Comment:
Can the database be passed so that the database will be the same instance where the collection is created or is gotten?